BackendNew/resources/openapi.yaml

```diff
@@ -277,7 +277,7 @@ paths:
                     type: string
         "404":
           description: students not found
-  /student/{student-id}/feedback:
+  /student/{student-id}/feedbacks:
```

Comment:
simple spelling mistake. It should be "feedback"

Reply:
It should be since it is a collection resource

Reply:
no word like "feedbacks"

Reply:
I will change it